### ISSUE TITLE
Failed Tests when building

### ISSUE THREAD

**Issue by ShoubhikRaj** [CalebFenton/simplify#44]
When building with gradle, the build fails because there were failed tests for `:simplify:test`. I am using Java SE 8u74 and gradle 2.11. The build report is attached with this post.
Build report :[index.pdf](https://github.com/CalebFenton/simplify/files/177231/index.pdf)

**Comment by CalebFenton** [comment]
This report doesn't say what the actual errors are, so I'm going to guess that the submodules were never cloned. Try cloning like this: `git clone --recursive https://github.com/CalebFenton/simplify.git`

Or run: `git submodule update --init --recursive`

If this doesn't fix it, try including some of the actual errors / stack traces.

**Comment by ShoubhikRaj** [comment]
It will take me some time to do the above task and give a proper feedback. Please keep this issue open. Thank you.

**Comment by ShoubhikRaj** [comment]
I have cloned the project using `git clone --recursive https://github.com/CalebFenton/simplify`. But still it gives the same error. So, I am uploading the images of command prompt when running the command `gradle build --stacktrace`. Take your time and read the images. By the way, I think that there should be pre-built jar releases for this project, just like there are releases for apktool.
![log1](https://cloud.githubusercontent.com/assets/17470159/14082108/db5e690e-f52b-11e5-9528-68f37cc118e7.PNG)
![log2](https://cloud.githubusercontent.com/assets/17470159/14082109/db626374-f52b-11e5-8a9e-44757b28cbe8.PNG)
![log3](https://cloud.githubusercontent.com/assets/17470159/14082123/ee2371e2-f52b-11e5-9d3a-5d2ea96983dd.PNG)
![log4](https://cloud.githubusercontent.com/assets/17470159/14082128/f2967a8a-f52b-11e5-99f2-b51890bd64b0.PNG)
![log5](https://cloud.githubusercontent.com/assets/17470159/14082133/f6d2885a-f52b-11e5-9f38-42454c8c6307.PNG)
![log6](https://cloud.githubusercontent.com/assets/17470159/14082136/f9e6bf8e-f52b-11e5-95ec-fbeb76fd700e.PNG)
![log7](https://cloud.githubusercontent.com/assets/17470159/14082137/fe4f453c-f52b-11e5-8187-26d61a463239.PNG)
![log8](https://cloud.githubusercontent.com/assets/17470159/14082140/017a5544-f52c-11e5-91f3-415bb7fbebb5.PNG)
![log9](https://cloud.githubusercontent.com/assets/17470159/14082142/037d7c0e-f52c-11e5-95db-3a79e2f72ab6.PNG)
![log10](https://cloud.githubusercontent.com/assets/17470159/14082144/05ad50ee-f52c-11e5-9295-63aa704e3169.PNG)
![log11](https://cloud.githubusercontent.com/assets/17470159/14082147/08c9a2be-f52c-11e5-87d6-bbc20ce44353.PNG)
![log12](https://cloud.githubusercontent.com/assets/17470159/14082150/0b30511a-f52c-11e5-84af-9d8b1e65c5ac.PNG)
![log13](https://cloud.githubusercontent.com/assets/17470159/14082154/0d77dba0-f52c-11e5-87dc-18e11ff0abcd.PNG)
![log14](https://cloud.githubusercontent.com/assets/17470159/14082159/0eba43b8-f52c-11e5-8614-31a0815645bd.PNG)

**Comment by CalebFenton** [comment]
I did a clean clone and wasn't able to reproduce the issue. It also builds fine on the CI servers, which are probably some variant of Linux. I'll try building on Windows later and see if that causes the problem.

In the future, it's better if you post text logs since they're indexable (searchable) and it's probably a lot less work than taking and uploading 10 screenshots.

Pre-built jar releases are located: https://github.com/CalebFenton/simplify/releases

**Comment by ShoubhikRaj** [comment]
I had to post 14 screenshots (I agree, its a tedious task) because when I tried to copy text from PowerShell, I found that the text had many parts missing.
Anyway, thank you for answering so quickly. I cannot test the build on Linux because I have only Windows 7 in my computer.
I think at this moment I will use the pre-built jars for my work. If I get Linux later, than I will try building.
I guess it is safe to close the issue now.
